hugo/layouts/langium/showcase-page.html

```diff
@@ -7,18 +7,58 @@
         height: 100%;
     }
 
+    .container {
+        width: 100%;
+        height: 100%;
+        border: 1px rgba(38, 136, 140) solid;
+        justify-content: center;
+        align-self: center;
+        display: flex;
```

Comment:
There's an issue with layout at pages that have a smaller height than expected, where monaco greatly exceeds the container bounds. (was using responsive design mode in Firefox, and pushed the page to a compact 2000x500)
![wide](https://user-images.githubusercontent.com/5070304/195790984-6760a7a4-02bd-4111-a9fe-2353e88df3ac.jpg)
You can hide the overflow like so, and monaco looks like it compensates just fine.
```suggestion
        display: flex;
        overflow: hidden;
```